.circleci/config.yml

```diff
@@ -15,14 +15,14 @@ cpu: &cpu
     image: default
   resource_class: medium
 
-gpu: &gpu
-  environment:
-    CUDA_VER: "10.2"
-    TORCH_CUDA_ARCH_LIST: "5.0;5.2;5.3"
-    TERM: xterm
-  machine:
-    image: ubuntu-1604:201903-01
-  resource_class: gpu.medium  # Tesla M60
+# gpu: &gpu
```

Comment:
quick check: are the circleCI changes expected in this PR?

Reply:
sorry, this was my fault.. Committed it to master without realising people will be adopting it to their feature branches. fixed :)